paris-processor/src/main/java/com/airbnb/paris/processor/android_resource_scanner/AndroidResourceId.kt

```diff
@@ -8,6 +8,8 @@ import com.squareup.javapoet.ClassName
 
 internal class AndroidResourceId(val value: Int, val className: ClassName, val resourceName: String) {
 
+    val rClassName: ClassName = className.topLevelClassName()
+
     val code: JavaCodeBlock = if (className.topLevelClassName() == AndroidClassNames.R) {
```

Comment:
You can replace usages of `className.topLevelClassName()` by `rClassName` here and maybe elsewhere if there are more.